chains/lisk/dispatcher_test.go

```diff
@@ -1,23 +1,87 @@
 package lisk_test
 
 import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"strconv"
 	"testing"
 
-	"github.com/sisu-network/deyes/chains/lisk"
+	"github.com/golang/protobuf/proto"
+	"github.com/sisu-network/deyes/chains/lisk/crypto"
+	ltypes "github.com/sisu-network/deyes/chains/lisk/types"
 	"github.com/sisu-network/deyes/types"
+
+	"github.com/sisu-network/deyes/chains/lisk"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	transactionResult    = "08021000180f200f322866323134643735626263346232656138396534333366336134356166383033373235343136656333"
+	defaultPassphrase    = "limit sort chase funny stumble stove diary regret demand march swim cycle"
+	defaultPrivateKey, _ = hex.DecodeString("fb3c93fbc5b28ef0879416d2928cefa8fc456a31f23302776b68f4481a5a1db837c8db9d10cd037f1fcbec7a9c091a86415e71975891b69f23bd60dcba233aa9")
+	defaultPublicKey, _  = hex.DecodeString("37c8db9d10cd037f1fcbec7a9c091a86415e71975891b69f23bd60dcba233aa9")
+	defaultAddress       = "lskgug634kfkpaatuushjpyvd8cucgfe8afsbsm8u"
+	moduleId             = uint32(2)
+	assetId              = uint32(0)
+	amount               = uint64(1000000000)
+	data                 = ""
+	fee                  = uint64(100000000)
+	recipientAddress, _  = hex.DecodeString("445f5ae1342837a1231f9d36d34a79145c1cd014")
+	networks             = map[string]string{"mainnet": "", "testnet": "e8832331820e5ba835012106a7c807b46c8b9c8672b6217b01373773fe87daf8"}
```

Comment:
This is an important variable. It should not be in  a test file since it will be used in the app. Also, can you check what is the value for mainnet? Just replace the API endpoint in the JS test file

Reply:
I asked lisk developer. Currently we can't connect to mainet via websocket

Reply:
4c09e6a781fc4c7bdb936ee815de8f94190f8a7519becd9de2081832be309a99 mainnet
15f0dacc1060e91818224a94286b13aa04279c640bd5d6f193182031d133df7c testnet